master/buildbot/process/build.py

```diff
@@ -89,6 +90,7 @@ def __init__(self, requests):
         self.terminate = False
 
         self._acquiringLock = None
+        self.result = SUCCESS   # overall result, may downgrade after each step
```

Comment:
In this file,
self.results is the list of step results, and self.result is the final build result.

We use the word results in plural everywhere else to refer to a single integer result.

I am not sure if results in english is more apropriate than result to store the result integer. Can the native reviewers provided a hint here?